lib/event-stream.js

```diff
@@ -269,4 +296,38 @@ EventStream.prototype._read = function() {
 	this.getLongPollInfo();
 };
 
+/**
+ * Implementation of stream-internal `_destroy` function (v8.0.0 and later).
+ * Called by stream consumers to effectively stop polling via the public
+ * `destroy()`.
+ * @returns {void}
+ * @private
+ */
+EventStream.prototype._destroy = function() {
+
+	if (!this.destroyed) {
+		/* istanbul ignore else */
+		if (typeof this.destroy === 'function') {
+			this.destroy();
+		} else {
+			// Node.js pre-v8.0.0 path; does some of what
+			// `Readable#destroy` in v8.0.0 does
+			process.nextTick(() => {
+				this.emit('close');
+			});
+			this.destroyed = true;
+		}
+	}
+	clearTimeout(this._retryTimer);
+	delete this._retryTimer;
+};
+
+/**
+ * Destroys this stream; for Node.js v8.0.0 and newer, just use
+ * `EventStream#destroy()`.
+ * @returns {void}
+ * @public
+ */
+EventStream.prototype.abort = EventStream.prototype._destroy;
```

Comment:
It's a bit undesirable to have two different methods to call depending on which version of Node.js the user is running.  Is it possible to check if `EventStream.prototype.destroy` already exists and conditionally add/polyfill it if it doesn't?

Reply:
yeah, we could do that.